usaspending_api/search/v2/views/spending_by_category_views/spending_by_industry_codes.py

```diff
@@ -70,7 +70,7 @@ def query_django(self, base_queryset: QuerySet) -> List[dict]:
                 row["name"] = fetch_psc_description_by_code(row["code"])
             elif self.industry_code_type == IndustryCodeType.NAICS:
                 row["id"] = None
-                row["name"] = fetch_naics_description_from_code(row["code"], row["name"])
+                row["name"] = fetch_naics_description_from_code(row["code"])
```

Comment:
Did you run into any cases where the `row["name"]` was being returned by this function? Seems removing this would change the current response value for Postgres implementation.

Reply:
you are absolutely right, not sure why i changed this